composer.json

```
@@ -11,21 +11,26 @@
         "facile.it",
         "coding standard"
     ],
+    "config": {
+        "sort-packages": true
+    },
     "scripts": {
-        "cs-check": "php-cs-fixer fix --dry-run --diff",
-        "cs-fix": "php-cs-fixer fix --diff"
+        "cs-check": "php-cs-fixer fix --dry-run --diff --allow-risky yes",
+        "cs-fix": "php-cs-fixer fix --diff --allow-risky yes"
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "ext-json": "*",
+        "composer-plugin-api": "^1.1 || ^2.0",
         "friendsofphp/php-cs-fixer": "^2.13.0",
-        "composer-plugin-api": "^1.1"
+        "symfony/console": "^3.0 || ^4.0 || ^5.1"
```

I would suggest to drop 3.0-3.3, it's non-LTS, unsupported versions: https://symfony.com/releases

Also, 5.0 should be supported too, since it's the same as 4.4.
```suggestion
        "symfony/console": "^3.4 || ^4.0 || ^5.0"
```